scripts/upgrade

```diff
@@ -74,46 +76,56 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
+#=================================================
+# CHECK THE PATH
+#=================================================
+
+path_url=$(ynh_normalize_url_path --path_url=$path_url)
+
 #=================================================
 # STANDARD UPGRADE STEPS
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
-ynh_print_info "Upgrading source files..."
 
-# Download, check integrity, uncompress and patch the source from app.src
-mkdir "$final_path/tmp"
-ynh_setup_source "$final_path/tmp"
-cp -a "$final_path/tmp/src/." "$final_path/."
-ynh_secure_remove "$final_path/tmp"
+if [ "$upgrade_type" == "UPGRADE_APP" ]
+then
+	ynh_script_progression --message="Upgrading source files..." --time --weight=1
```

Comment:
```suggestion
	ynh_print_info --message="Upgrading source files..."
```

Or change each `ynh_print_info` to `ynh_script_progression` with the right weight ;-)

Reply:
Thanks, I missed this one ^^